templates/header.war.html

```diff
@@ -1,73 +1,10 @@
-{% extends "base.html" %}
+<!DOCTYPE html>
 
-{% block distribution_instruction %}
+<html lang="en">
 
-<p>
-  This is the WAR package repository of {{ product_name }} for installation.
-<p>
-{% endblock %}
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="refresh" content="1; url='https://www.jenkins.io/doc/book/installing/war-file/'" />
+</head>
 
-{% block java_instructions %}
-  <p>
-    You will need to explicitly install a supported Java runtime environment (JRE),
-    e.g. <a href="https://adoptium.net/">Eclipse Temurin</a>.
-  </p>
-
-  <h2>
-    Weekly Release Line
-  </h2>
-
-  <p>
-    Supported Java versions for the weekly release line are:
-  </p>
-
-  <dl>
-    <dt>2.419 (August 2023) and newer</dt>
-    <dd>Java 11, Java 17, or Java 21</dd>
-
-    <dt>2.357 (June 2022) and newer</dt>
-    <dd>Java 11 or Java 17</dd>
-
-    <dt>2.164 (February 2019) and newer</dt>
-    <dd>Java 8 or Java 11</dd>
-
-    <dt>2.54 (April 2017) and newer</dt>
-    <dd>Java 8</dd>
-
-    <dt>1.612 (May 2015) and newer</dt>
-    <dd>Java 7</dd>
```

Comment:
Where is this information on https://www.jenkins.io/doc/book/installing/war-file/?